### ISSUE TITLE
[charter] describe deliverables

### ISSUE THREAD

**Issue by samuelweiler** [w3c/editing#294]
I'm used to seeing the list of deliverables in charters include a brief summary of each document, e.g. stolen from the doc's abstract. I particularly note the lack of that in the "Other Deliverables" section, where we don't have links to the documents.

I'm also aware that our charters are too wordy to begin with. Up to you whether to fix that for the existing deliverables, but I do think more words are needed for the "other deliverables", unless you have links available for not-yet-adopted proposed docs.

**Comment by siusin** [comment]
@gked @johanneswilm 
Is there a public archive or a link to the SpellcheckAPI that we can add to the charter please?

**Comment by johanneswilm** [comment]
@siusin I am not sure which one that would be. Someone started a community WG recently around that. Is that the one you are thinking of?  https://github.com/w3c/document-services

I would be all for that effort to become part of the Editing WG. I am not entirely sure why they have decided to form a community WG rather than join the existing Editing TF. I have tried to engage with them and given feedback on their initial proposal.

**Comment by johanneswilm** [comment]
@siusin I see a spellchecker API came up on the TPAC 2019 meeting: https://lists.w3.org/Archives/Public/public-editing-tf/2019Oct/0004.html . But as far as I can tell, @BoCupp-Microsoft was going to investigate more as there were some security concerns and it was not clear whether we could move beyond those. I don't think I have seen a draft spec since then. Whether or not JS can get access to the browser internal spell checking dictionaries really depends on whether or not it is possible to overcome the security concerns. The creation of the API itself does not seem that complex.
